Troika/Extensions/UIView+Troika.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public extension UIView {
+    public convenience init(withAutoLayout autoLayout: Bool) {
```

Comment:
This is convenient. Keep in mind that several of the components is designed with a custom initializer and won't be able to use this (PreviewGridView, MarketGridView, Button, Ribbon, TextField, ToastView).